### ISSUE TITLE
KeyListener with FlightControllerKey.HOME_LOCATION only called once

### ISSUE THREAD

**Issue by reddigfabian** [dji-sdk/Mobile-SDK-Android#199]
I am setting up `KeyListeners` for all `DJIKey`s that my app currently cares about. When setting up a listener for `FlightControllerKey.HOME_LOCATION` I only receive a call to `onValueChange(...)` once, when initially connecting the aircraft.

Out of curiosity, I set up listeners for `HOME_LOCATION_LATITUDE` and `HOME_LOCATION_LONGITUDE` and both of these are triggered properly each time the aircraft takes off for a new flight. This makes me think there is something wrong with the code for value changes in `FlightControllerKey.HOME_LOCATION`.

SDK Version: 4.3.2
DJI Product: Phantom 4 Advanced
Android Version: 5.0.2
Android Device: Nexus 7 Tablet (2013)
Android Studio Version: 3.0

**Comment by Michael-DJI** [comment]
value of FlightControllerKey.HOME_LOCATION will be updated only when you setting a different value.
this key is a replacement of these two methods:
"setHomeLocation()"  and "getHomeLocation()" in FlightController

**Comment by Djacob502** [comment]
I also ran into this. The home location is set when first turning on the drone. However the home location is then reset, or set a second time, shortly after takeoff. It is this second time that reddigfabian is referencing that is not captured by FlightControllerKey.HOME_LOCATION. From your response it looks like this is the desired behavior. 
Since knowing the home location is quite important - the solution is to set up listeners for HOME_LOCATION_LATITUDE/LONGITUDE to check when the home location changes. These two listeners will always return any changes in the home location no matter how the home location was changed. 
We should only use key.HOME_LOCATION for those times when we program the home location to be changed.
